docs/adapting-settings.md

```diff
@@ -95,7 +95,7 @@ This is the allowed resources extensions list that user can upload.
 
 ### RESOURCES_FILE_ALLOWED_DOMAINS
 
-**default**: `tuple()`
+**default**: `list()`
```

Comment:
Maybe `[]` instead of `list()`. `tuple()` is used because empty tuple is written `(, )` which isn't very readble